### PR TITLE
Pin the right version of Pweave per Python version

### PR DESCRIPTION
**Pull request by ccordoba12**

### CODE REVIEW

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,8 @@ requirements:
   run:
     - python
     - spyder >=3.2
-    - pweave
+    - pweave 0.25  # [py27]
```
Comment (goanpeca) [inline recipe/meta.yaml:27]: @ccordoba12, is not it better to use `pweave <=0.25` ? it has to be that specific version?

Comment (ccordoba12) [reply recipe/meta.yaml:27]: There are no previous versions of Pweave here on conda-forge, and that's the last version to work with Python 2.

Comment (goanpeca) [reply recipe/meta.yaml:27]: Ok!

**Pull request merged**